comet/dumper.py

```diff
@@ -35,22 +38,26 @@ def __del__(self):
         if self.lock_file is not None:
             os.remove(self.lock_file)
 
-    async def _new_file(self):
+    async def _new_file(self, redis):
```

Comment:
I don't understand why you need redis for controlling file access here?

Reply:
The dumper needs to start a new file once in a while to let the archiver have the last one. Since there's a dumper in each process, they need to synchronize information about when one of the processes switches to a new file.